README.md

```diff
@@ -1,4 +1,4 @@
-[![Neon](https://user-images.githubusercontent.com/13738772/236813940-dcfdcb5b-69d3-449b-a686-013febe834d4.png)](https://neon.tech)
+[![Neon](https://github.com/neondatabase/neon/assets/10074684/6268f966-4b70-4f92-b694-8aca60f8abd7)](https://neon.tech)
```

Comment:
The image doesn't redirect to neon.tech anymore, it just opens the picture in a full-screen.

https://github.com/neondatabase/neon/blob/7a3bbecaef039bfad3ef8cbc2d490d9fcad25803/README.md

Reply:
Thank you. I'll see if I can figure this out.